cse-runner-app/src/test/java/com/farao_community/farao/cse/runner/app/services/PiSaServiceTest.java

```diff
@@ -23,51 +29,130 @@
 @SpringBootTest
 class PiSaServiceTest {
 
-    @Autowired
-    private PiSaConfiguration piSaConfiguration;
-
     @Autowired
     private PiSaService piSaService;
 
     @Test
-    void testCheckPiSaWithMissingElement() {
-        String networkFilename = "20210901_2230_test_network_pisa_uncomplete_model_test.uct";
+    void testPiSaPreProcessInIdccWithOneLinkInSetpoint() {
+        String networkFilename = "20210901_2230_test_network_pisa_test_one_link_connected_and_setpoint.uct";
         Network network = Importers.loadNetwork(networkFilename, getClass().getResourceAsStream(networkFilename));
 
-        assertThrows(CseDataException.class, () -> piSaService.isPiSaPresent(network));
+        assertEquals(1000, piSaService.getPiSaLink1Processor().getFrFictiveGenerator(network).getTargetP(), 1);
```

Comment:
I think it's better to declare the value of the delta as a constant since we use the same value (1) several times.